### ISSUE TITLE
Drag space is greyed out, even when the window is unmaximized

### ISSUE THREAD

**Issue by nollinvoyd** [Aris-t2/CustomCSSforFx#131]
Win 10 Pro, Fx 61.0, CustomCSSforFx 1.9.5

Fresh unaltered profile, only the unaltered contents of "CustomCSSforFx-master\classic folder" in the Chrome folder.  I've using CustomCSSforFx since v1.93, experiencing the same behavior.

**Comment by Aris-t2** [comment]
"Drag-space" is only available, if the "Titlebar" is disabled. Both options are unrelated to this project.

**Comment by nollinvoyd** [comment]
My apologies.  Honest mistake.  I saw that drag space was disabled, so I created a fresh profile to test it.  

The drag space option box was available to be checked.  I moved my chrome folder with the default CustomCSSforFx to the profile, after which I noticed drag space was disabled.

After I moved the chrome folder, I must have enabled the menubar.  It never occurred to me that enabling the menubar would disable the drag space option, but that's what happened.

Check it out.  Doesn't make sense to me, and a google search brings up no mention of it from what I can see.
